### ISSUE TITLE
Site editor: iterate on site icon animation

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#65428]
[While this initial PR](https://github.com/WordPress/gutenberg/pull/63986#) landed to try to clarify that the site icon is a back button, I think it's missing the mark and looks more broken than anything else. @WordPress/gutenberg-design can we iterate here? I think we can do better for 6.7. At the least, I think we should revert to what was there before. If we can, we should iterate.

https://github.com/user-attachments/assets/7d76eba6-bacb-47ea-baf1-957cecc25a52

**Comment by paaljoachim** [comment]
This looks like a bug. As something similar shows up example when there is a missing icon when using FontAwesome.
![Hover top left site logo in Site Editor](https://github.com/user-attachments/assets/68105593-8d4c-42f9-accf-ed46c160e679)

**Comment by afercia** [comment]
I'd totally agree, see my previous comment at https://github.com/WordPress/gutenberg/pull/64161#issuecomment-2320255399

Still the entire mechanism of going back and exiting the editor is unclear and needs some rethinking together with the meaning and usage of the WP logo / Site Icon preview. See https://github.com/WordPress/gutenberg/issues/57813
